### PR TITLE
feat: change descriptions, add exception

### LINKED ISSUE #82: Web Interface | Already used address situation
**Is your feature request related to a problem? Please describe.**
Ever since i’ve created my IPFS node, grapheneX can’t use the default port of 8080 for its web interface ` An error occurred: [Errno 98] Address already in use` _(Might want to correct this typo in Error btw)_
That's indeed where my IPFS is setup at, this can easily be fixed by using another port by using the [host:port] argument.
**Describe the solution you'd like**
Have a prompt of grapheneX when setting him to start its web interface at a specific address and/or port, to ask to save this address as the default one next. Can easily be changed by default once again, by using -localhost:8080 or maybe with the inclusion of a -default argument
**Describe alternatives you've considered**
I’ve considered suggesting to auto-save it, but a user might want to use a specific port for only specific occasions.
I’ve also thought of having grapheneX have a backup default address that it would try if the default one was already used, but this might be causing some issues if it actually bothers with the user web setup.
Same goes, for trying a random port, or even worse, scanning all the available ones.
**Additional context**
However, as grapheneX modules are habitually ¨Setup once, and forget about it¨, I suppose this isn’t really needed.

### PR DESCRIPTION
**Pull request by jvirgovic**
- Added more detailed error message if the default addr:port is already in use (closes #82)
- Changed description for: Auto_Logout, Set_File_Permissions
- Added myself as contributor :)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by jvirgovic** [comment]
By logging more detailed error message I believe #82 can be closed

**Comment by jvirgovic** [comment]
Actually, I edited the error message to be more generic, and not say that default localhost:8080 is already in use, as the user might've specified completely different addr:port that could also already be in use by some different service.

LMK what you think

File: .all-contributorsrc [hunk 1]
```diff
@@ -109,6 +109,17 @@
...
+    {
+      "login": "jxd1337",
+      "name": "jxd1337",
+      "avatar_url": "https://avatars.githubusercontent.com/u/138708600?v=4",
+      "profile": "https://github.com/jxd1337",
+      "contributions": [
+        "code",
+        "ideas",
+        "doc"
+      ]
```
Comment (orhun) [inline .all-contributorsrc]: I think this should be automated - I recall mentioning the bot would automatically create a PR. See #69 

So this change is not really needed :)

File: graphenex/core/web/__init__.py [hunk 2]
```diff
@@ -53,5 +54,8 @@ def run_server(args=None, exit_shell=True):
         socketio.stop()
         if exit_shell:
             Shell().do_exit(None)
+    except socket.error as e:
+        if e.errno == 98:
+            logger.error('Looks like the specified addr:port is already in use!')
```
Comment (orhun) [inline graphenex/core/web/__init__.py]: Can we maybe use an error message like "`localhost:8000 is already in use!`" to show whichever address/port is being used?

**Comment by orhun** [comment]
@all-contributors please add @jxd1337 for code, ideas, doc

**Comment by jvirgovic** [comment]
error message edited to display addr:port correctly

**Review by orhun — APPROVED** [review]

**Pull request merged**